server/test/index.js

```diff
@@ -236,3 +236,13 @@ describe('POST /api/v1/users/request', () => {
       });
   });
 });
+describe('GET /api/v1/users/request', () => {
+  it('Should return 200 for succesfully getting all requests', (done) => {
+    chai.request(app)
+      .get(`${requestURL}`)
+      .end((err, res) => {
+        expect(res.body).to.be.an('object');
```

Comment:
What is the content of the object I am expecting? 
I guess it has message and status. . . test for all those please @OKiMaureen